utils/kata-manager.sh

```diff
@@ -254,14 +254,17 @@ pre_checks()
 {
 	info "Running pre-checks"
 
+	local only_kata="${1:-}"
```

Comment:
I would prefer calling this as "skip_containerd" instead of "only_kata" with the default value being false.

Reply:
Only kata is what it is called way back when the cli options are parsed. Do you want me to renaime it all the way back, or just in pre-checks?

Reply:
I would rename it everywhere.

Reply:
like that?